src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -30,141 +34,177 @@ const RegistrationForm: FunctionComponent<Props> = ({
   const { t } = useTranslation();
 
   return (
-    <div className={styles.registrationForm}>
-      <Formik
-        initialValues={initialValues}
-        onSubmit={values => {
-          setFormValues(values);
-          try {
-            submitChild({
-              variables: {
-                birthdate: values.child.birthdate,
-                firstName: values.child.firstName,
-                lastName: values.child.lastName,
-                guardianFirstName: values.guardian.firstName,
-                guardianLastName: values.guardian.lastName,
-                email: values.guardian.email,
-              },
-            });
-          } catch (err) {
-            // TODO: Error handling.
-            // eslint-disable-next-line no-console
-            console.error(err);
+    <div className={styles.registrationFormContainer}>
+      <div className={styles.registrationForm}>
+        <Formik
+          initialValues={initialValues}
+          initialErrors={
+            (!initialValues.agree && {
+              agree: validateRequire(''),
+            }) ||
+            {}
           }
-        }}
-      >
-        {({ values, handleChange, isSubmitting, handleSubmit, isValid }) => (
-          <form onSubmit={handleSubmit}>
-            <div className={styles.childInfo}>
-              <div className={styles.childName}>
+          onSubmit={values => {
+            setFormValues(values);
+            try {
+              submitChild({
+                variables: {
+                  birthdate: values.child.birthdate,
```

Comment:
Do we have a separate issue for adding the new fields to the queries towards the backend?

Reply:
I dont do that cause im not sure if backend support or not. But issue is welcome i think